src/stylesheets/abstracts/_variables.scss

```diff
@@ -17,9 +17,11 @@ $v-globals: (
   
   /// Colors
   text-clr: #212121,
+  hover-text-clr: #fff,
```

Comment:
We already have `bg-clr`.

Reply:
yes, but I think it's better to be able to separately change the color of the text for the hover

Reply:
Ok, it makes sense